### PR TITLE
Update uwp6.2 subscriptions to listen for updated ProjectN builds

### PR DESCRIPTION
**Pull request by zamont**

### CODE REVIEW

**Comment by zamont** [comment]
@tommcdon @dagood

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -322,7 +322,8 @@
     {
       "triggerPaths": [
         "https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/release/uwp6.2/Latest.txt",
-        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/coreclr/release/uwp6.2/Latest.txt"
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/coreclr/release/uwp6.2/Latest.txt",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/buildtools/master/Latest.txt"
```
Comment (dagood) [inline Maestro/subscriptions.json]: This doesn't match https://github.com/dotnet/coreclr/blob/release/uwp6.2/dependencies.props currently, do you have changes planned there? (Same difference in CoreFX with https://github.com/dotnet/corefx/blob/release/uwp6.2/dependencies.props.)

release/uwp6.1 dependencies.props appear to be set up ok, for reference.

Comment (zamont) [reply Maestro/subscriptions.json]: I still need to update those...let me finish those two repos and I'll swing back around to this PR

**Comment by zamont** [comment]
uwp-setup and CoreCLR have been updated correctly.

Waiting for https://github.com/dotnet/corefx/pull/30886 to close which updates the dependencies.props that matches this subscriptions.json

**Comment by dagood** [comment]
I'm going to go ahead and merge this optimistically. :smile: (Nothing dangerous happens if there's a mismatch between triggers and configuration, and it'll be nice to get this out of the way in case nothing needs to change.)

**Pull request merged**